Lib/idlelib/colorizer.py

```diff
@@ -215,7 +222,7 @@ def recolorize(self):
             t0 = time.perf_counter()
             self.recolorize_main()
             t1 = time.perf_counter()
-            if DEBUG: print("%.3f seconds" % (t1-t0))
+            if DEBUG: print("%.3f seconds" % (t1 - t0))
```

Comment:
Spaces are not required or even wrong.  "c = (a + b) * (a - b)" is labelled 'wrong'.